modules/micromenu/micromenu.lua

```diff
@@ -647,6 +648,8 @@ function module:SetMicromenuExtraButtons()
 			})
 			buttonMiddle:SetBackdropColor(module:RGB("Micromenu"))
 			buttonMiddle:SetBackdropBorderColor(0, 0, 0, 0)
+			print("micro background hide")
```

Comment:
Looks like two of these bad boys were left in, other than that LGTM.

Using IsVisible is definitely a more sensible solution.